docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -3,14 +3,71 @@
 
 NOTE: The UI settings service is available both server and client side.
 
-=== Server side usage
+=== Overview
+
+UI settings are configurable from the Advanced Settings page in Management and control the behavior of {kib}. uiSettings are stored in a config saved object and, as such, conform to the same conditions as other <<saved-objects-service, Saved Objects>>.
+
+There are several ways to configure an advanced setting:
```

Comment:
I've converted the bullet points into a sub-index for guiding readers through the contents on this page.